tests/test_content_events.py

```
@@ -0,0 +1,330 @@
+class SystemMessage:
+    content = "lifecycle"
+
+
+class UnknownObject:
+    def __repr__(self) -> str:
+        return "UnknownObject(raw-secret-like-value)"
+
+
+class ExplodingMessage:
+    @property
+    def content(self) -> Any:
```

Ruff (0.16.2)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `content`

(ANN401)